docker-compose_tests.yml

```diff
@@ -14,6 +14,7 @@ services:
     environment:
       - AUTHSERVER=http://auth:8089/common
       - ISSUER=auth:8089
+      - AUDIENCE=test
```

Comment:
This part belongs in the check audience commit

Reply:
true, same with change in integration_tests.py